builtins/amp-video.js

```diff
@@ -36,7 +36,31 @@ export function installVideo(win) {
     }
 
     /** @override */
+    prerenderAllowed() {
+      return true;
+    }
+
+    /** @override */
+    buildCallback() {
+      /** @private */
+      this.realPreloadAttrValue_ = this.element.getAttribute('preload');
+      const posterAttr = this.element.getAttribute('poster');
+
+      if (!posterAttr) {
+        console/*OK*/.warn(
+            'No "poster" attribute has been provided for amp-video.');
+      }
+    }
+
+    /** @override */
     layoutCallback() {
+      const docState = documentStateFor(this.getWin());
+      // Disable video preload in prerender mode and reset it when rendered.
+      if (docState.getVisibilityState() === 'prerender') {
+        this.element.setAttribute('preload', 'none');
```

Comment:
As far as I understood the layoutCallback will be called again when the document is actually rendered, no? 

I guess it would be better to set it on the video instead of changing it on the element and then propagating it. Will update that.